orchagent/portsorch.cpp

```diff
@@ -2357,10 +2355,19 @@ void PortsOrch::initPortCapLinkTraining(Port &port)
     SWSS_LOG_WARN("Unable to get %s LT support capability", port.m_alias.c_str());
 }
 
-bool PortsOrch::isFecModeSupported(const Port &port, sai_port_fec_mode_t fec_mode)
+bool PortsOrch::isFecModeSupported(const Port &port, sai_port_fec_mode_t fec_mode, bool override_fec)
 {
     initPortSupportedFecModes(port.m_alias, port.m_port_id);
 
+    if (!override_fec && !fec_override_sup)
+    {
+        return false;
+    }
```

Comment:
@dgsudharsan suggest to move this check outside at the caller and log a print "Skipping FEC override setting for fec = auto-fec" and not change the meaning of this function which is not related to fec supported or not.

Reply:
Done